### PR TITLE
Renames NT Rep to NT Consultant, moves NT Rep title to alt titles

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Renames all occurrences of "Nanotrasen Representative" to "Nanotrasen Consultant".

Made on host request.

## Changelog
:cl:
tweak: "Nanotrasen Representative" has been renamed to "Nanotrasen Consultant".
/:cl:

### CODE REVIEW

**Comment by Zonespace27** [comment]
What was the reason behind requesting this change?

**Comment by tf-4** [comment]
> What was the reason behind requesting this change?

@hugmeorelse

**Comment by AtoriBirb** [comment]
I understand the why of it but it still seems like punishing the role for the antics of bad faith players. Kinda a common thing, we should just hold people playing it to a higher standard.

**Comment by Deek-Za** [comment]
I 100% get the heart behind this. The NT rep gives a bit more power because they seemingly represent the will of NT. But the NT rep is fallible to a degree. I don't think we *have* to push a name change. I think we just have to push the ruleset better and discourage people from using it inappropriately. 
But also I feel like I may or may not have been involved in this change request from a sour apple of a round.
This is a case of, I think it's fine as is, and we can just all strive to do/enforce it better.
But honestly it's just a small name change at the end of the day so it won't matter in the long term. (See: "As a consultant of NT...."
edit: But also someone just said something about the authoritative angle losing it's edge with this change. So maybe.. it's not so far gone of a conclusion?

**Comment by GoldenAlpharex** [comment]
I liked the sound of NT Rep.

**Comment by Zonespace27** [comment]
Name-wise, I prefer representative over consultant

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
-	outfit = /datum/outfit/job/nanotrasen_representative
-	plasmaman_outfit = /datum/outfit/plasmaman/nanotrasen_representative
+	outfit = /datum/outfit/job/nanotrasen_consultant
+	plasmaman_outfit = /datum/outfit/plasmaman/nanotrasen_consultant
 
 	paycheck = PAYCHECK_HARD
 	paycheck_department = ACCOUNT_CMD
 
-	display_order = JOB_DISPLAY_ORDER_NANOTRASEN_REPRESENTATIVE
+	display_order = JOB_DISPLAY_ORDER_nanotrasen_consultant
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm]: RegEx user detected.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
This was a quick review, I probably missed something.

**Comment by hugmeorelse** [comment]
> I understand the why of it but it still seems like punishing the role for the antics of bad faith players. Kinda a common thing, we should just hold people playing it to a higher standard.

It's not "bad faith" players, considering this is only for Veteran Crew. Misuse of this will get you removed from Veteran Crew extremely quick.

This role has no control over Station Command. They have always been a consultant, acting as the bridge between Central Command and Station Command, and the title has always been misleading. **This corrects that mistake.** It removes any sort of ambiguity of what their role is about.

**Comment by tf-4** [comment]
Forgot icon states, will take me a moment to hunt them down.

**Comment by kannthus** [comment]
I feel its a little unnecessary, but at the same time if the names are staying as alt titles I don't think it's the end of the world. I fear newer players might misunderstand "Consultant" written on an airlock as the term for a medical personnel room, having it say "NT Consultant" rather then just "Consultant's room" will help with the ambiguity

File: modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm [hunk 2]
```diff
@@ -99,8 +99,8 @@
 /datum/job/mime
 	alt_titles = list("Mime", "Pantomimist")
 
-/datum/job/nanotrasen_representative
-	alt_titles = list("Nanotrasen Representative", "Nanotrasen Diplomat", "Central Command Representative")
+/datum/job/nanotrasen_consultant
+	alt_titles = list("Nanotrasen Consultant", "Nanotrasen Diplomat", "Central Command Consultant", "Nanotrasen Representative")
```
Comment (Zonespace27) [inline modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm]:
```suggestion
	alt_titles = list("Nanotrasen Consultant", "Nanotrasen Diplomat", "Central Command Consultant", "Nanotrasen Representative", "Central Command Representative")
```
May as well?

**Comment by tf-4** [comment]
Can't sit here and resolve merge conflicts while it's in limbo anymore. Feel free to fix the conflict and merge it, or don't.

**Comment by Gandalf2k15** [comment]
Rip TOM.

**Comment by ghost** [comment]
this broke the hud icon :dying:

**Pull request merged**